JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -65,10 +67,20 @@ module Microsoft.ApplicationInsights {
         public session: Context.Session;
 
         /**
+        * Custom properties added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public properties;
+
+        /**
+        * Custom measurements added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public measurements;
```

Comment:
Do we want to define measurements in `TelemetryContext` knowing that they are supported only by a small subset of telemetry types?